x-pack/plugins/stack_alerts/server/alert_types/geo_threshold/alert_type.ts

```diff
@@ -177,40 +172,7 @@ export interface GeoThresholdParams {
   boundaryIndexQuery?: Query;
 }
 
-export function getAlertType(
-  logger: Logger
-): {
-  defaultActionGroupId: string;
-  actionGroups: ActionGroup[];
-  executor: ({
-    previousStartedAt: currIntervalStartTime,
-    startedAt: currIntervalEndTime,
-    services,
-    params,
-    alertId,
-    state,
-  }: {
-    previousStartedAt: Date | null;
-    startedAt: Date;
-    services: AlertServices;
-    params: GeoThresholdParams;
-    alertId: string;
-    state: AlertTypeState;
-  }) => Promise<AlertTypeState>;
-  validate?: {
-    params?: {
-      validate: (object: unknown) => GeoThresholdParams;
-    };
-  };
-  name: string;
-  producer: string;
-  id: string;
-  actionVariables?: {
-    context?: ActionVariable[];
-    state?: ActionVariable[];
-    params?: ActionVariable[];
-  };
-} {
```

Comment:
:+1:  on the clean up

Reply:
This PR is the first part of the licensing implementation. This issue should cover the UI https://github.com/elastic/kibana/issues/85282